### PR TITLE
Exercise showing results of expressions typed at the prompt are automatically displayed while in script mode, the print function has to be used to evaluate the expression.

### PR DESCRIPTION
**Pull request by mbengtanyi**
atically displayed while in script mode, the print function has to be used to evaluate the expression.

### CODE REVIEW

File: Tessy/Chapter_1/exercise_1.12.6 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
+[GCC 5.4.0 20160609] on linux
+Type "copyright", "credits" or "license()" for more information.
```
Comment (ch3nkula) [inline Tessy/Chapter_1/exercise_1.12.6]: You forgot this part: `Type 6 + 4 * 9 at the Python prompt and hit enter. Record what happens.`. I was expecting to see you type the expression on the terminal first before moving on with executing the script.

**Comment by ch3nkula** [comment]
Why did you close the PR @mbengtanyi ?

**Comment by mbengtanyi** [comment]
It was an error.

**Comment by ch3nkula** [comment]
@mbengtanyi, you can reopen it!

**Comment by ch3nkula** [comment]
Check the text box at the bottom of the pate, there is a button that say "Reopen Pull request". Thanks

**Pull request merged**